test/e2e/framework/kcp.go

```diff
@@ -513,11 +513,27 @@ func (c *kcpServer) monitorEndpoint(client *rest.RESTClient, endpoint string) {
 			return
 		}
 		if err != nil {
-			c.t.Errorf("error contacting %s: %v", endpoint, err)
+			c.t.Errorf("error contacting %s: %v", endpoint, unreadyComponentsFromError(err))
 		}
 	}, 1*time.Second)
 }
 
+// there doesn't seem to be any simple way to get a metav1.Status from the Go client, so we get
+// the content in a string-formatted error, unfortunately.
+func unreadyComponentsFromError(err error) string {
```

Comment:
please give some example output and what you make out of it. This is hard to follow.

Reply:
Example here: https://github.com/kcp-dev/kcp/pull/1488#issuecomment-1180866176

This happens in like every single e2e test output, did you not notice these big blobs? These are the tricky ones to read when some prober fails unexpectedly ...